### ISSUE TITLE
Authors multiselect issue

### ISSUE THREAD

**Issue by damisul** [abartov/bybeconv#113]
While fixing https://github.com/abartov/bybeconv/issues/46
I faced following problem.

Currently if any filter is applied system will display in authors selection popup only persons related to filtered works.

But I believe such behaviour introduce additional issues:
1) If we need to select several authors and forgot to pick one from the list (e.g. we picked 4 out of 5 desired authors) we are not able to select him by opening popup again. The reason is that when we narrowed list of works of 4 authors, popup will only display persons related to selected works (i.e. those 4 authors only). We will need to reset filter and pick all 5 authors again.

2) Second issue is caused by the fact that this popup actually shows all authors and translators related to filtered works.
So we can pick 4 authors, but popup will show more persons, including co-authors and translators. This behaviour was very surprising to me and I find it non-intuitive.
Another issue related to this is when we apply some filter by author. E.g. we need only female authors, this popup will show mostly women, but will also include few male names, because they are co-authors or translators.

Not sure how to address this properly.
One of possible ways could be to always display full list of authors.
Or maybe we can fetch list of authors from db, but take in account additional filters like gender or author name.

**Comment by abartov** [comment]
I agree these two examples are usability issues. But I, too, am not sure what would be better.  

Given hundreds of total authors and the desire to filter by period, or gender, etc., it does seem to me to be valuable to only offer relevant authors (i.e. ones matching the other filters).

I will draw our UX designer's attention to this.
